easy_my_coop_website/controllers/main.py

```diff
@@ -256,29 +256,30 @@ def validation(self, kwargs, logged, values, post_file):
             redirect = "easy_my_coop_website.becomecompanycooperator"
             email = kwargs.get("company_email")
 
-        if (
-            "g-recaptcha-response" not in kwargs
-            or kwargs["g-recaptcha-response"] == ""
-        ):
-            values = self.fill_values(values, is_company, logged)
-            values.update(kwargs)
-            values["error_msg"] = _(
-                "the captcha has not been validated,"
-                " please fill in the captcha"
-            )
+        if request.website.recaptcha_key_site:
```

Comment:
I would rather have it like a boolean field defined on res_company because if they forget to define the key nothing prevent it.

Reply:
@cesarlr what do you think?

Reply:
Ok, good idea!